src/Plugins/SimplnxCore/src/SimplnxCore/Filters/ReadCSVFileFilter.cpp

```diff
@@ -67,6 +67,22 @@ enum class IssueCodes
   IGNORED_TUPLE_DIMS = -200
 };
 
+StringVector RemoveIllegalCharacters(StringVector& headers)
+{
+  for(int i = 0; i < headers.size(); i++)
+  {
+    auto& headerName = headers[i];
+    // Replace all illegal characters with '_' character. The header names become array names which is the issue.
+    // This should have been taken care of in the GUI, but if someone is trying this from Python they will not have done that
+    // or if they are just reading it in through nxrunner.
+    headerName = StringUtilities::replace(headerName, "&", "_");
+    headerName = StringUtilities::replace(headerName, ":", "_");
+    headerName = StringUtilities::replace(headerName, "/", "_");
+    headerName = StringUtilities::replace(headerName, "\\", "_");
+    headerName = StringUtilities::replace(headerName, "\"", "");
```

Comment:
```suggestion
    headerName = StringUtilities::replace(headerName, "\"", "_");
```

I think you missed an underscore here.

Reply:
@joeykleingers 
This was a conscious decision on my part. If you have a header that looks like
```
"Some:Header"
```

Then we end up with _Some_Header_ which really isn't what they user is going to want. I would expect almost every user to want to just remove those quotes.